config/routes.rb

```diff
@@ -25,6 +25,7 @@ def put(*_args)
 
       resources :requests, :only => %i(index show) do
         resources :stages, :only => [:index]
+        post :actions, :action => 'create', :controller => 'requests'
```

Comment:
why the syntax is different from workflows/id/requests or templates/id/workflows? shouldn't you use actions controller?